packages/web/components/discover/discover-filter-from.component.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Form, Select, DatePicker, Slider, Button, Checkbox } from 'antd';
+import { DiscoverFilterSectionComponent } from './discover-filter-section.component';
+import {
+  GetDiscoverQueryVariables,
+  useGetLanguagesQuery,
+  useGetGenresQuery,
+} from '../../utils/graphql';
+
+interface DiscoverFilterFormComponentProps {
+  params: GetDiscoverQueryVariables;
+  onFinish: (formParams: GetDiscoverQueryVariables) => void;
+}
+export function DiscoverFilterFormComponent(
+  props: DiscoverFilterFormComponentProps
+) {
+  const languagesQuery = useGetLanguagesQuery();
+  const genresQuery = useGetGenresQuery();
+
+  const TMDBLanguages = languagesQuery.data?.languages;
+  const TMDBMovieGenres = genresQuery.data?.genres.movieGenres;
+  const [year, setYear] = useState<number>(); // with adding Moment library this state can be moved inside of forms
+  const [form] = Form.useForm();
+
+  const options = useMemo(
+    () =>
+      TMDBLanguages?.map((l) => (
+        <Select.Option key={l.code} value={l.code}>
+          {l.language}
+        </Select.Option>
+      )),
+    [TMDBLanguages]
+  );
+
+  const onYearChange = useCallback((_, value: string) => {
```

Comment:
I had to bring this function out of form because Ants return me a Moment date but I need string. any suggestion to fix it? was wondering about Moment library.

Reply:
Yes you should be able to replace moment with dayjs which is already used and much smaller https://ant.design/docs/react/replace-moment